config/initializers/saml_idp.rb

```diff
@@ -45,3 +45,5 @@
     ServiceProvider.new(issuer_or_entity_id).metadata
   end
 end
+
+VALID_AUTHNCONTEXTS = JSON.parse(Figaro.env.valid_authncontexts).freeze
```

Comment:
could we keep the namespacing here, in `Saml::Idp::Constants::` ? I dislike adding global vars.